lib/util/config.js

```diff
@@ -0,0 +1,6 @@
+const Configstore = require('configstore');
```

Comment:
Here `const` is used instead of `var` (which is the better way IMO), therefore I suggest doing the same in the other files OR using here `var` on behalf of consistency.